app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -392,6 +395,17 @@ fun MapScreen(
   val errorMessageIdState = profileViewModel.errorMessageId.collectAsState()
   val profileState = profileViewModel.profile.collectAsState()
 
+  var button1Bounds by remember { mutableStateOf<Rect?>(null) }
+  var button2Bounds by remember { mutableStateOf<Rect?>(null) }
+  var shortText by remember { mutableStateOf(false) }
+  LaunchedEffect(button1Bounds, button2Bounds) {
+    if (button1Bounds != null &&
+        button2Bounds != null &&
+        button1Bounds!!.overlaps(button2Bounds!!)) {
+      shortText = true
+    }
+  }
+
```

Comment:
Question: I understand how that works but what would happen if the device is even smaller so that having a `shortText` is not small enough?

Reply:
It would overlap again.

I agree that its a valid concern, but imo that's not really worth the effort of adding an even smaller button. The small phone from the Android Studio emulator is already extremely small. It would probably be so small that we would also start getting issues with the localisation button. However, if you think its appropriate, I can surely add a third size which just says like "Search" or just has a magnifying glass.

Reply:
Looks good like this! You don't have to do any more job for this.